.github/workflows/neuron-ci.yml

```diff
@@ -7,7 +7,6 @@ concurrency:
 on:
   push:
     branches: 
-      - master
```

Comment:
We have several deliverables coming from this (codecov report, wheels, documentation push). Sometimes we merge several PRs one after the other without updating, so still useful.

Reply:
I don't touch those ones